src/dtgtk/togglebutton.c

```diff
@@ -162,7 +140,6 @@ GtkWidget *dtgtk_togglebutton_new(DTGTKCairoPaintIconFunc paint, gint paintflags
   button->canvas = gtk_drawing_area_new();
   gtk_container_add(GTK_CONTAINER(button), button->canvas);
   dt_gui_add_class(GTK_WIDGET(button), "dt_module_btn");
-  gtk_widget_set_name(GTK_WIDGET(button), "dt-toggle-button");
```

Comment:
Do not remove it. Still needed on some places to allow different settings between dt-button and dt-toggle-button, like in blending tabs